packages/sdk/src/conversation/command.ts

```diff
@@ -40,11 +63,43 @@ export class CommandBot {
   /**
    * Registers commands into the command bot.
    *
-   * @param commands The command to registered.
+   * @param commands The commands to register.
    */
   public registerCommands(commands: TeamsFxBotCommandHandler[]): void {
     if (commands) {
       this.middleware.commandHandlers.push(...commands);
     }
   }
+
+  /**
+   * Registers a sso command into the command bot.
+   *
+   * @param command The command to register.
+   */
+  public registerSsoCommand(ssoCommand: TeamsFxBotSsoCommandHandler): void {
+    if (ssoCommand) {
+      if (!this.middleware.getActivityHandler()) {
+        this.middleware.setActivityHandler(new DefaultSsoExecutionActivityHandler(this.ssoConfig));
+      }
+      this.middleware.commandHandlers.push(ssoCommand);
+      this.middleware.activityHandler?.addCommand(ssoCommand);
+    }
+  }
+
+  /**
+   * Registers commands into the command bot.
+   *
+   * @param commands The commands to register.
+   */
+  public registerSsoCommands(ssoCommands: TeamsFxBotSsoCommandHandler[]): void {
+    if (ssoCommands) {
+      if (!this.middleware.getActivityHandler()) {
+        this.middleware.setActivityHandler(new DefaultSsoExecutionActivityHandler(this.ssoConfig));
+      }
+      for (const ssoCommand of ssoCommands) {
+        this.middleware.activityHandler?.addCommand(ssoCommand);
```

Comment:
seems this.middleware.activityHandler is exist, if not, will set in line 97
when will meet this.middleware.activityHandler get null?

Reply:
Yes, that's true, however, typescript cannot detect this, I need to use ? to avoid it complains about activityHandler may be null.